build_tools/azure/posix-docker.yml

```diff
@@ -44,6 +45,22 @@ jobs:
       ${{ insert }}: ${{ parameters.matrix }}
 
   steps:
+    - bash: |
+        set -ex
+        # Check if run only specific tests but on all random seeds
+        # commit message must take the form
+        # <title> [all random seeds]
+        # <test_name_1>
+        # <test_name_2>
+        # ...
+        if [[ "$COMMIT_MESSAGE" =~ \[all\ random\ seeds\] ]]; then
+          message=$(echo $COMMIT_MESSAGE | sed -n -e 's/^.*all\ random\ seeds\] //p')
```

Comment:
I think unix commands harder to maintain in the long run. Can we parse the commit message in a simple Python script?